File: Armando_Lorea/Java/Practica1/src/main/java/com/poo/sis401s/objeto/Practica1.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.poo.sis401s.objeto;

/**
 *
 * @author arman
 */
public class Practica1 {

    public static void main(String[] args) {
        Persona persona = new Persona();
        
        persona.setNombre("Daniela");
        persona.setEdad(25);
        persona.setColor("Moreno");
        persona.setTamano(1.55);
        persona.setPeso(62.0);
        
        System.out.println("Nombre: " + persona.getNombre());
        System.out.println("Edad: " + persona.getEdad());
        System.out.println("Color: " + persona.getColor());
        System.out.println("Tamaño: " + persona.getTamano());
        System.out.println("Peso: " + persona.getPeso());
        }
}

File: Paola_Colin/Python/Practica1.py
class Persona:
    def __init__(self, nombre="", edad=0, color="", tamano=0.0, peso=0.0):
        self.nombre = nombre
        self.edad = edad
        self.color = color
        self.tamano = tamano
        self.peso = peso

persona = Persona("Daniela", 28, "Moreno", 1.54, 62.0)

print("Nombre:", persona.nombre)
print("Edad:", persona.edad)
print("Color:", persona.color)
print("Tamano:", persona.tamano)
print("Peso:", persona.peso)